src/messenger.js

```diff
@@ -178,14 +177,14 @@ export class Messenger {
    */
   sendStartCommand(args) {
     let channel = null;
-    if (this.acceptsPort_ && typeof this.win_.MessageChannel == 'function') {
+    if (this.acceptsChannel_ && typeof this.win_.MessageChannel == 'function') {
       channel = new this.win_.MessageChannel();
     }
     if (channel) {
       this.sendCommand('start', args, [channel.port2]);
       // It's critical to switch to port messaging only after "start" has been
       // sent. Otherwise, it won't be delivered.
-      this.switchToPort_(channel.port1);
+      this.switchToChannel_(channel.port1);
```

Comment:
nit
Can we name port1 better. 
something like initialPort? or something like that which indicates that it is the first port to be connected.

Reply:
Unfortunately, `port1` is not our name :( It comes from MessageChannel API.